packages/commonwealth/client/scripts/views/pages/user_dashboard/TrendingCommunitiesPreview/CommunityPreviewCard/CommunityPreviewCard.tsx

```diff
@@ -0,0 +1,70 @@
+import clsx from 'clsx';
+import { pluralize } from 'helpers';
+import React from 'react';
+import { CWIcon } from 'views/components/component_kit/cw_icons/cw_icon';
+import CommunityInfo from '../../../../../models/ChainInfo';
+import { CWCard } from '../../../../components/component_kit/cw_card';
+import { CWCommunityAvatar } from '../../../../components/component_kit/cw_community_avatar';
+import { CWText } from '../../../../components/component_kit/cw_text';
+import './CommunityPreviewCard.scss';
+
+type CommunityPreviewCardProps = {
+  community?: CommunityInfo;
+  monthlyThreadCount?: number;
+  isCommunityMember?: boolean;
+  hasUnseenPosts?: boolean;
+  onClick?: () => any;
+  isExploreMode?: boolean;
+};
+
+const CommunityPreviewCard = ({
+  community = {} as any,
```

Comment:
Even though this gets instantiated right away, we can still type it as `CommunityInfo`